cmake/SpectreGetGitHash.cmake

```diff
@@ -15,15 +17,17 @@ if(EXISTS ${CMAKE_SOURCE_DIR}/.git)
       OUTPUT_VARIABLE GIT_HASH
       OUTPUT_STRIP_TRAILING_WHITESPACE
       )
+    set(GIT_BRANCH_COMMAND "${GIT_EXECUTABLE} rev-parse --abbrev-ref HEAD")
     execute_process(
-      COMMAND ${GIT_EXECUTABLE} rev-parse --abbrev-ref HEAD
+      COMMAND bash -c "${GIT_BRANCH_COMMAND}"
       WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
       OUTPUT_VARIABLE GIT_BRANCH
       OUTPUT_STRIP_TRAILING_WHITESPACE
       )
+    set(GIT_DESCRIPTION_COMMAND "${GIT_EXECUTABLE} describe \
+--always --first-parent --match v[0-9]* HEAD")
```

Comment:
This looks fragile since this is a pretty general glob and things will break if there are any files starting with "v[0-9]" in whatever directory this runs in.  Does the CMake syntax allow quoting here?  (Single quotes are fine.)

Reply:
seems to work with single quotes